modules/core/src/lib/attribute/data-column.js

```diff
@@ -216,7 +216,7 @@ export default class DataColumn {
       this.value = opts.value;
 
       // Copy the type of the buffer into the accessor
-      accessor.type = buffer.accessor.type;
+      accessor.type = opts.type || buffer.accessor.type;
```

Comment:
Once `Buffer` is created, `type` is fixed isn't it? why do we have to overwrite this? Is this covering non external buffer case?

Reply:
`buffer.accessor.type` is inferred from the data type array that is used to populate it by luma, but the chunk of memory can be interpreted as any data type in WebGL. This is needed if you use interleaved buffer with mixed data types.